prototypes/metallurgy.lua

```diff
@@ -468,23 +468,22 @@ data:extend(
   
   --------------- ZINC PROCESSING ---------------
 	
-	-- SEPARATED COPPER AND ZINC CONCENTRATE
+	-- 1) ZINC CONCENTRATE (FROTH FLOATATION)
```

Comment:
This will probably need some work in general, the PR is based from the github version rather than the forum version, so it doesnt use subsidiary ore here yet.